packages/dm-core-plugins/src/index.tsx

```diff
@@ -7,9 +7,9 @@ import { JobInputEditPlugin } from './job/JobInputEditPlugin'
 import { JobControlPlugin } from './job/JobControlPlugin'
 import { FormPlugin } from './form/FormPlugin'
 import HeaderPlugin from './header/HeaderPlugin'
-import { DefaultPreviewComponent } from './default-preview/DefaultPreviewComponent'
-import { DefaultPdfComponent } from './default-pdf/DefaultPdfComponent'
-import { EditBlueprint } from './blueprint/EditBlueprint'
+import JsonPlugin from './json/JsonPlugin'
+import { PdfPlugin } from './pdf/PdfPlugin'
+import { BlueprintPlugin } from './blueprint/EditBlueprint'
 import ExplorerPlugin from './explorer/ExplorerPlugin'
```

Comment:
A detail really, but ideally we should be consistent in how we export plugins too.

Reply:
true!
creating new issue since to avoid merge conflicts https://github.com/equinor/dm-core-packages/issues/124